hack/windows/Prepare-Node.ps1

```diff
@@ -123,9 +149,9 @@ nssm set kubelet DependOnService docker
 New-NetFirewallRule -Name kubelet -DisplayName 'kubelet' -Enabled True -Direction Inbound -Protocol TCP -Action Allow -LocalPort 10250
 
 # Create netadapter for kube-proxy, the default full name of the adapter is "vEthernet (HNS Internal NIC)"
-& ./Prepare-ServiceInterface.ps1
+& .\Prepare-ServiceInterface.ps1 -InterfaceAlias "HNS Internal NIC"
```

Comment:
will -InterfaceAlias "HNS Internal NIC" needed in both CRI

Reply:
yes , needed in both cases